transport/http/inbound_example_test.go

```diff
@@ -83,3 +84,52 @@ func ExampleMux() {
 	}
 	// Output: hello from /health
 }
+
+func ExampleInterceptor() {
+	// import nethttp "net/http"
```

Comment:
Is this a part of examples?

Reply:
Copied from the other examples in this file - guessing it's to clarify that `nethttp` is coming from `net/http`.

Reply:
I think Grayson wants it to be obvious in the code example where the "nethttp" is coming from.